hnn_core/network.py

```diff
@@ -14,6 +14,54 @@
 from .params import create_pext
 
 
+def read_spikes(fname, gid_dict=None):
```

Comment:
This function should be added to the `api.rst` page: https://45-168215891-gh.circle-artifacts.com/0/html/api.html. Maybe add a subsection called input/output?